### PR TITLE
[awattar] Fix price handler refresh

### LINKED ISSUE #17027: [aWATTar] After 4.2. update the hourly prices do not refresh
## Expected Behavior
After the update from openhab 4.1 stable to openhab 4.2. stable the hourly prices get the most recent prices from the aWATTar API.

## Current Behavior
After the update from openhab 4.1 stable to openhab 4.2. stable the hourly prices do not refresh and the thing remains in status "unknown". The openhab.log does not state any warns or errors. The uninstallation and installation of the add-on did not change anything.

## Possible Solution
unknown

## Steps to Reproduce (for Bugs)
1. configure aWATTar Bridge with Land: AT, Ust-Satz: 20 and Basispreis: 1,5
2. update from openhab 4.1 stable to openhab 4.2

## Your Environment
runtimeInfo:
  version: 4.2.0
  buildString: Release Build
locale: de-AT
systemInfo:
  configFolder: /etc/openhab
  userdataFolder: /var/lib/openhab
  logFolder: /var/log/openhab
  javaVersion: 17.0.9
  javaVendor: Azul Systems, Inc.
  javaVendorVersion: Zulu17.46+19-CA
  osName: Linux
  osVersion: 6.8.0-1006-raspi
  osArchitecture: aarch64
  availableProcessors: 4
  freeMemory: 207981056
  totalMemory: 970981376
  uptime: 72776
  startLevel: 100
addons:
  - automation-jsscripting
  - binding-automower
  - binding-awattar
  - binding-doorbird
  - binding-fronius
  - binding-harmonyhub
  - binding-homeconnect
  - binding-hue
  - binding-icalendar
  - binding-knx
  - binding-mqtt
  - binding-network
  - binding-openweathermap
  - binding-systeminfo
  - binding-telegram
  - binding-unifi
  - misc-openhabcloud
  - persistence-influxdb
  - persistence-mapdb
  - transformation-jsonpath
  - transformation-map
  - transformation-scale
  - transformation-xpath
  - transformation-xslt
  - ui-basic
  - voice-picotts

### PR DESCRIPTION
**Pull request by J-N-K**
Fixes #17027 

Should be backported to 4.2.x

### CODE REVIEW

File: bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarPriceHandler.java [hunk 1]
```diff
@@ -90,7 +90,8 @@ public void initialize() {
                  * here
                  */
                 thingRefresher = scheduler.scheduleAtFixedRate(this::refreshChannels,
-                        getMillisToNextMinute(1, timeZoneProvider), THING_REFRESH_INTERVAL, TimeUnit.SECONDS);
+                        getMillisToNextMinute(1, timeZoneProvider), THING_REFRESH_INTERVAL * 1000,
+                        TimeUnit.MILLISECONDS);
```
[the remark this answers is not in the document]
Comment (jlaur) [reply bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarPriceHandler.java:94]: Regression of #16871, the other unit is milliseconds (`getMillisToNextMinute`).

**Review by jlaur — APPROVED** [review]
Thanks!

File: bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarPriceHandler.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (lsiepel) [reply bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarPriceHandler.java:94]: Ah offcourse, thanks.

**Comment by aMU5Ed** [comment]
Here is the debug log from the 4.2.0 release awattar binding during a bundle restart:

`2024-07-09 20:19:18.529 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Querying state active
2024-07-09 20:19:18.531 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Deactivating component
2024-07-09 20:19:18.532 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : registration change queue [unregistered]
2024-07-09 20:19:18.567 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Querying state active
2024-07-09 20:19:18.568 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : getting deactivate: deactivate
2024-07-09 20:19:18.569 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Locating method deactivate in class org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory
2024-07-09 20:19:18.569 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Declared Method org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory.deactivate([interface org.osgi.service.component.ComponentContext]) not found
2024-07-09 20:19:18.570 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Locating method deactivate in class org.openhab.core.thing.binding.BaseThingHandlerFactory
2024-07-09 20:19:18.571 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Found deactivate method: protected void org.openhab.core.thing.binding.BaseThingHandlerFactory.deactivate(org.osgi.service.component.ComponentContext)
2024-07-09 20:19:18.572 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : invoking deactivate: deactivate: parameters [org.apache.felix.scr.impl.manager.ComponentContextImpl]
2024-07-09 20:19:18.572 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : invoked deactivate: deactivate
2024-07-09 20:19:18.573 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : DependencyManager: osgi.ds.satisfying.condition close component unbinding from org.apache.felix.scr.impl.manager.ComponentContextImpl@50e31048 at tracking count 3 refpairs: [[RefPair: ref: [{org.osgi.service.condition.Condition}={service.id=6, service.bundleid=0, service.scope=singleton, service.pid=0.org.osgi.service.condition.ConditionImpl, osgi.condition.id=true}] service: [null]]]
2024-07-09 20:19:18.573 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : DependencyManager: $001 close component unbinding from org.apache.felix.scr.impl.manager.ComponentContextImpl@50e31048 at tracking count 2 refpairs: [[RefPair: ref: [{org.openhab.core.i18n.TranslationProvider, org.openhab.core.i18n.LocaleProvider, org.openhab.core.i18n.LocationProvider, org.openhab.core.i18n.TimeZoneProvider, org.openhab.core.i18n.UnitProvider}={location=48.305395, 16.151830, service.id=304, service.bundleid=156, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.internal.i18n.I18nProviderImpl, language=de, service.config.label=Regional Settings, measurementSystem=SI, component.id=260, timezone=Europe/Vienna, service.config.category=system, region=AT, service.config.description.uri=system:i18n, service.pid=[org.openhab.i18n, org.openhab.i18n]}] service: [org.openhab.core.internal.i18n.I18nProviderImpl@5d88416]]]
2024-07-09 20:19:18.574 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : DependencyManager: $000 close component unbinding from org.apache.felix.scr.impl.manager.ComponentContextImpl@50e31048 at tracking count 1 refpairs: [[RefPair: ref: [{org.openhab.core.io.net.http.HttpClientFactory, org.openhab.core.io.net.http.WebSocketFactory}={service.id=199, service.bundleid=182, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.io.net.http.internal.WebClientFactoryImpl, component.id=49}] service: [org.openhab.core.io.net.http.internal.WebClientFactoryImpl@6997dd8e]]]
2024-07-09 20:19:18.574 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Unset and deconfigured implementation object for component in deleteComponent for reason Unspecified
2024-07-09 20:19:18.575 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Changed state from active to satisfied
2024-07-09 20:19:18.576 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Deactivating dependency managers
2024-07-09 20:19:18.577 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Disabling dependency managers
2024-07-09 20:19:18.578 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : unregistering service listener for dependency $000
2024-07-09 20:19:18.578 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : unregistering service listener for dependency $001
2024-07-09 20:19:18.579 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : unregistering service listener for dependency osgi.ds.satisfying.condition
2024-07-09 20:19:18.579 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Querying state satisfied
2024-07-09 20:19:18.580 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Changed state from satisfied to disposed
2024-07-09 20:19:18.580 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(457)] : Disposing component (reason: 6)
2024-07-09 20:19:18.591 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : BundleComponentActivator : ComponentHolder created.
2024-07-09 20:19:18.593 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Dependency Manager created $000interface=org.openhab.core.io.net.http.HttpClientFactory, filter=null, policy=static, cardinality=1..1, bind=null, unbind=null, updated=null, field=null, field-option=null, collection-type=service, parameter=0
2024-07-09 20:19:18.594 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Dependency Manager created $001interface=org.openhab.core.i18n.TimeZoneProvider, filter=null, policy=static, cardinality=1..1, bind=null, unbind=null, updated=null, field=null, field-option=null, collection-type=service, parameter=1
2024-07-09 20:19:18.595 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Dependency Manager created osgi.ds.satisfying.conditioninterface=org.osgi.service.condition.Condition, filter=(osgi.condition.id=true), policy=dynamic, cardinality=1..1, bind=null, unbind=null, updated=null, field=null, field-option=null, collection-type=null, parameter=null
2024-07-09 20:19:18.596 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Component created: DS=DS14, implementation=org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory, immediate=false, default-enabled=true, factory=null, configuration-policy=optional, activate=activate, deactivate=deactivate, modified=null configuration-pid=[binding.awattar]
2024-07-09 20:19:18.596 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Component Services: scope=singleton, services=[org.openhab.core.thing.binding.ThingHandlerFactory]
2024-07-09 20:19:18.597 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Component Properties: {osgi.ds.satisfying.condition.target=(osgi.condition.id=true)}
2024-07-09 20:19:18.598 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Querying state disabled
2024-07-09 20:19:18.598 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Querying state disabled
2024-07-09 20:19:18.599 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Component can not be activated since it is in state disabled
2024-07-09 20:19:18.600 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory] : Querying state disabled
2024-07-09 20:19:18.601 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Updating target filters
2024-07-09 20:19:18.602 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : No change in target property for dependency $000: currently registered: false
2024-07-09 20:19:18.603 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] :  No existing service listener to unregister for dependency $000
2024-07-09 20:19:18.603 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Setting target property for dependency $000 to null
2024-07-09 20:19:18.604 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : New service tracker for $000, initial active: false, previous references: {}, classFilter: (objectClass=org.openhab.core.io.net.http.HttpClientFactory), initialReferenceFilter (objectClass=org.openhab.core.io.net.http.HttpClientFactory)
2024-07-09 20:19:18.605 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $000 tracker reset (closed)
2024-07-09 20:19:18.606 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $000 tracking 1 SingleStatic added {org.openhab.core.io.net.http.HttpClientFactory, org.openhab.core.io.net.http.WebSocketFactory}={service.id=199, service.bundleid=182, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.io.net.http.internal.WebClientFactoryImpl, component.id=49} (enter)
2024-07-09 20:19:18.607 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $000 tracking 1 SingleStatic active: false trackerOpened: false optional: false
2024-07-09 20:19:18.608 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $000 tracking 1 SingleStatic added {org.openhab.core.io.net.http.HttpClientFactory, org.openhab.core.io.net.http.WebSocketFactory}={service.id=199, service.bundleid=182, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.io.net.http.internal.WebClientFactoryImpl, component.id=49} (exit)
2024-07-09 20:19:18.608 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $000 tracker opened
2024-07-09 20:19:18.609 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : registering service listener for dependency $000
2024-07-09 20:19:18.610 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : No change in target property for dependency $001: currently registered: false
2024-07-09 20:19:18.611 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] :  No existing service listener to unregister for dependency $001
2024-07-09 20:19:18.612 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Setting target property for dependency $001 to null
2024-07-09 20:19:18.612 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : New service tracker for $001, initial active: false, previous references: {}, classFilter: (objectClass=org.openhab.core.i18n.TimeZoneProvider), initialReferenceFilter (objectClass=org.openhab.core.i18n.TimeZoneProvider)
2024-07-09 20:19:18.613 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $001 tracker reset (closed)
2024-07-09 20:19:18.614 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $001 tracking 2 SingleStatic added {org.openhab.core.i18n.TranslationProvider, org.openhab.core.i18n.LocaleProvider, org.openhab.core.i18n.LocationProvider, org.openhab.core.i18n.TimeZoneProvider, org.openhab.core.i18n.UnitProvider}={location=48.305395, 16.151830, service.id=304, service.bundleid=156, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.internal.i18n.I18nProviderImpl, language=de, service.config.label=Regional Settings, measurementSystem=SI, component.id=260, timezone=Europe/Vienna, service.config.category=system, region=AT, service.config.description.uri=system:i18n, service.pid=[org.openhab.i18n, org.openhab.i18n]} (enter)
2024-07-09 20:19:18.615 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $001 tracking 2 SingleStatic active: false trackerOpened: false optional: false
2024-07-09 20:19:18.617 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $001 tracking 2 SingleStatic added {org.openhab.core.i18n.TranslationProvider, org.openhab.core.i18n.LocaleProvider, org.openhab.core.i18n.LocationProvider, org.openhab.core.i18n.TimeZoneProvider, org.openhab.core.i18n.UnitProvider}={location=48.305395, 16.151830, service.id=304, service.bundleid=156, service.scope=bundle, osgi.ds.satisfying.condition.target=(osgi.condition.id=true), component.name=org.openhab.core.internal.i18n.I18nProviderImpl, language=de, service.config.label=Regional Settings, measurementSystem=SI, component.id=260, timezone=Europe/Vienna, service.config.category=system, region=AT, service.config.description.uri=system:i18n, service.pid=[org.openhab.i18n, org.openhab.i18n]} (exit)
2024-07-09 20:19:18.617 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm $001 tracker opened
2024-07-09 20:19:18.618 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : registering service listener for dependency $001
2024-07-09 20:19:18.619 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] :  No existing service listener to unregister for dependency osgi.ds.satisfying.condition
2024-07-09 20:19:18.619 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Setting target property for dependency osgi.ds.satisfying.condition to (osgi.condition.id=true)
2024-07-09 20:19:18.620 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : New service tracker for osgi.ds.satisfying.condition, initial active: false, previous references: {}, classFilter: (objectClass=org.osgi.service.condition.Condition), initialReferenceFilter (&(objectClass=org.osgi.service.condition.Condition)(osgi.condition.id=true))
2024-07-09 20:19:18.621 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm osgi.ds.satisfying.condition tracker reset (closed)
2024-07-09 20:19:18.622 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm osgi.ds.satisfying.condition tracking 3 SingleDynamic added {org.osgi.service.condition.Condition}={service.id=6, service.bundleid=0, service.scope=singleton, service.pid=0.org.osgi.service.condition.ConditionImpl, osgi.condition.id=true} (enter)
2024-07-09 20:19:18.623 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm osgi.ds.satisfying.condition tracking 3 SingleDynamic added {org.osgi.service.condition.Condition}={service.id=6, service.bundleid=0, service.scope=singleton, service.pid=0.org.osgi.service.condition.ConditionImpl, osgi.condition.id=true} (exit)
2024-07-09 20:19:18.624 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : dm osgi.ds.satisfying.condition tracker opened
2024-07-09 20:19:18.624 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : registering service listener for dependency osgi.ds.satisfying.condition
2024-07-09 20:19:18.625 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Changed state from disabled to unsatisfiedReference
2024-07-09 20:19:18.626 [DEBUG] [ternal.handler.AwattarHandlerFactory] - bundle org.openhab.binding.awattar:4.2.0 (327)[org.openhab.binding.awattar.internal.handler.AwattarHandlerFactory(458)] : Component enabled
`

**Comment by aMU5Ed** [comment]
Please provide me a binary backported to 4.2. release build that I can test it.

**Pull request merged**